vue-app/src/graphql/queries/GetRecipients.graphql

```diff
@@ -1,5 +1,5 @@
-query GetRecipients {
-  recipients {
+query GetRecipients($registryAddress: String!) {
+  recipients(where: { recipientRegistry: $registryAddress }) {
```

Comment:
Filter the recipients by the currently used recipient registry. We don't want recipients from other registries.